### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by fruizga**
Commit for checks the pull request

### CODE REVIEW

File: api/v1/app.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from models import storage
+from api.v1.views import app_views
+from os import getenv
+
+
+app = Flask(__name__)
+
+app.register_blueprint(app_views, url_prefix="/api/v1")
+
+@app.teardown_appcontext
```
Comment (andreammgcol) [inline api/v1/app.py:14]: Can you please add description on this method?

File: api/v1/views/index.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python3
+
+from api.v1.views import app_views
+from flask import jsonify
+
+
+@app_views.route('/status', strict_slashes=False)
+def status():
+    return jsonify({'status': "OK"})
```
Comment (andreammgcol) [inline api/v1/views/index.py:9]: can you please add description for this code?

Comment (fruizga) [reply api/v1/views/index.py:9]: Of course, I'll do this changes in the next commit.

File: api/v1/app.py (hunk 1 above)
Comment (fruizga) [reply api/v1/app.py:14]: Of course, I'll do this changes in the next commit.

**Review by andreammgcol — APPROVED** [review]

**Pull request merged**